bigframes/streaming/__init__.py

```diff
@@ -24,9 +25,11 @@
 
 def to_bigtable(
     query: str,
+    *,
     instance: str,
     table: str,
-    bq_client: Optional[bigquery.Client] = None,
+    service_account: Optional[str] = None,
```

Comment:
Maybe `service_account_email`? I'm worried `service_account` is too ambiguous. I don't want someone to think they need a key file, JSON, for example.

Reply:
FWIW: `service_account_email` would be consistent with google-auth. See: https://googleapis.dev/python/google-auth/latest/reference/google.auth.compute_engine.html

Reply:
Makes sense, changed, thanks.